atom/browser/api/atom_api_web_contents.cc

```diff
@@ -689,6 +690,28 @@ void WebContents::PrintToPDF(const base::DictionaryValue& setting,
       PrintToPDF(setting, callback);
 }
 
+void WebContents::AddWorkSpace(
+    const base::FilePath& path,
+    const JsCompletionCallback& callback) {
+  if (path.empty()) {
```

Comment:
Shouldn't we need `v8::Locker` here before using `isolate()`? I'm not sure, @zcbenz might has advices.